plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -170,40 +198,32 @@ var FbListInlineEdit = new Class({
 			}
 			if (typeOf(this.td.getPrevious()) === 'element') {
 				e.stop();
-				this.select(e, this.td.getPrevious());
+				this.select(e, this.getPreviousEditable(this.td));
 			}
 			break;
 		case 40:
 			//down
-			if (this.inedit) {
-				return;
+			if (this.inedit && this.options.tabSave) {
```

Comment:
Hmmm. Maybe the tab option (and the up/down arrows) should only work on elements that are input type=text (when in edit mode, otherwise it would be OK). But like I said, the tab option was always there. 

Or maybe users should just be aware that the 'Save on tab" feature is intended for use in lists that only have simple text inputs? But It does make the inlineedit work "much like a spreadsheet", as advertised in the Wiki. (Which is what sold me on even trying Fabrik to begin with. I NEED this feature.)